relay-server/src/services/processor/span/processing.rs

```diff
@@ -590,6 +590,31 @@ fn normalize(
 
     populate_ua_fields(span, user_agent.as_deref(), client_hints.as_deref());
 
+    // INP spans sets some top level span attributes inside span.data so make sure to pull
+    // them out to the top level before further processing.
+    if let Annotated(Some(ref mut data), _) = span.data {
+        if let Some(exclusive_time) = match data.other.get("sentry.exclusive_time") {
```

Comment:
Could you add `sentry.exclusive_time` and `profile_id` as fields on the `SpanData` type? I assume a hypothetical SDK that can only send Otel spans would still send these as span attributes? Or are these legacy fields that future SDKs should not send?

Reply:
They can continue sending them as span attributes.